<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="computationalMethodGroupBox">
      <property name="title">
       <string>Алгоритмический метод</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QSpinBox" name="amountOfNumbersToGenerateWithComputationalMethod">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="generateRandomNumbersWithComputationalMethodPushButton">
         <property name="text">
          <string>Сгенерировать</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="NumbersTableWidget" name="computationalMethodRandomNumbersTableWidget"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="tableMethodGroupBox">
      <property name="title">
       <string>Табличный метод</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <widget class="QSpinBox" name="amountOfNumbersToGenerateWithTableMethod">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="generateRandomNumbersWithTableMethodPushButton">
         <property name="text">
          <string>Сгенерировать</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="NumbersTableWidget" name="tableMethodRandomNumbersTableWidget"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NumbersTableWidget</class>
   <extends>QTableWidget</extends>
   <header>numbers_table_widget.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
